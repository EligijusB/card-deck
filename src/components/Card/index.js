import "./index.scss";

const Card = ({title, src}) => {
    return(
        <img className="card" src={src} alt={title} />
    );
};

export default Card;