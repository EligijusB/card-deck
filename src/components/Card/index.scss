img.card {
    width: 80px;
    height: 100px;
    padding: 5px;
}