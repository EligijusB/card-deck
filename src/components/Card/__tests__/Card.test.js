import renderer from "react-test-renderer";
import Card from "..";

describe("<Card />", () => {
    const props = {
        title: "Test Card",
        src: "http://someImage.jpeg"
    };

    it("Should render correctly", () => {
        const comp = renderer.create(<Card {...props} />);
        expect(comp.toJSON()).toMatchSnapshot();
    });

});