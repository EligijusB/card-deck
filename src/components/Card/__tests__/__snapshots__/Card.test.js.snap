// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> Should render correctly 1`] = `
<img
  alt="Test Card"
  className="card"
  src="http://someImage.jpeg"
/>
`;
